_data/entertainment.yml

```diff
@@ -171,6 +171,8 @@ websites:
       sms: Yes
       software: Yes
       doc: http://blog.twitch.tv/2015/11/two-factor-authentication-now-available-on-your-twitch-account/
+      exceptions:
+          text: "To activate 2-Step Verification, you must provide a mobile phone number."
```

Comment:
Please change `2-Step Verification` to `2FA` or `two factor authentication`.
2SV and 2FA are different. See [Contributing.md](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#a-note-on-definitions) for more info